include/vara/Feature/Feature.h

```diff
@@ -34,9 +35,11 @@ class Feature {
   FeatureListType Alternatives;
   RelationshipListTy Relationships;
   bool Opt;
+  std::optional<std::unique_ptr<Location>> Loc;
```

Comment:
We don't need to put the location on the heap, especially not when we copy it anyhow when calling `getLocation`